server/src/services/users.ts

```diff
@@ -159,3 +159,29 @@ export const getUserPhotosById = async (userId: string) => {
 	const userPhotos = await getPhotosByUserId(userId);
 	return userPhotos;
 };
+
+export const checkCompletnessByUserId = async (userId: string) => {
+	if ((await userHasPhotos(userId)) && (await userDataIsNotNULL(userId))) return true;
+	return false;
+};
+
+// export const getUserCompletenessById = async (userId: string) => {
+// 	const completeness = await getCompletenessByUserId(userId);
+// 	if (!completeness) throw new AppError('No user with provided id', 400);
+// 	if (!completeness.complete){
+// 		completeness.complete = await checkCompletnessByUserId(userId);
+// 	}
+
+// 	return completeness;
+// };
+
+export const getUserCompletenessById = async (userId: string) => {
```

Comment:
I think there's reason to rename that one into getAndUpdateUserCompletnessById, cause we're making some changes on the way here